sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -60,9 +61,22 @@ def get_field_value(field, value, read_result):  # pylint: disable=too-many-retu
             key: FormField._from_generated(key, value, read_result)
             for key, value in value.value_object.items()
         }
+    if value.type == "selectionMark":
+        return value.text
+
     return None
 
 
+class SelectionMarkState(str, Enum):
```

Comment:
I have in .NET `FormSelectionMarkState` because of the type `FormSelectionMark`
but I have to say I am not a fan of the long name for state. @annelo-msft was also saying that this (the state) could be one of those types where we don't need the whole name as it can be implied.
What do you think?

Reply:
Less opinionated here since this type is more for documentation. I can align to `FormSelectionMarkState`

Reply:
I think `SelectionMarkState` could work since it is a property on `FormSelectionMark` so using prefix `Form` would be redundant.

Reply:
Yeah I am ok with that too. I will remove the `Form` in .NET